### ISSUE TITLE
can not install on windows

### ISSUE THREAD

**Issue by mtfcd** [rest-nvim/rest.nvim#463]
log from lazy.nvim:
```
 rest.nvim 📂 http 
        C:\Users\cc\scoop\apps\lua51\current\lua5.1.exe: cannot open e  -e io.write(_VERSION:sub(5)) : No such file or directory
        C:\Users\cc\scoop\apps\lua51\current\lua5.1.exe: cannot open e  -e io.write(_VERSION:sub(5)) : No such file or directory

        Error: Lua 5.1 interpreter not found at C:/Users/cc/scoop/apps/lua/current

        Please set your Lua interpreter with:

           luarocks --lua-version=5.1 --local config variables.LUA <d:\path\lua.exe>
        Failed installing rest.nvim with `luarocks`.

        --------------------------------------------------------------------------------

        Trying to build from source.
        C:\Users\cc\scoop\apps\lua51\current\lua5.1.exe: cannot open e  -e io.write(_VERSION:sub(5)) : No such file or directory
        C:\Users\cc\scoop\apps\lua51\current\lua5.1.exe: cannot open e  -e io.write(_VERSION:sub(5)) : No such file or directory

        Error: Lua 5.1 interpreter not found at C:/Users/cc/scoop/apps/lua/current

        Please set your Lua interpreter with:

           luarocks --lua-version=5.1 --local config variables.LUA <d:\path\lua.exe>
```

I hava those installed use scoop:
```
lua                 5.4.7-2       main       2024-09-15 19:18:04
lua-for-windows     5.1.5-52      main       2024-09-13 00:40:11
lua-language-server 3.10.5        main       2024-08-25 15:21:46
lua51               5.1.5         versions   2024-09-15 19:19:19
luarocks            3.11.1        main       2024-09-15 19:18:05
```

lua51 is installed at:
```
    Directory: C:\Users\cc\scoop\apps\lua51\5.1.5

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---            2014/5/6     3:54          17920 lua5.1.exe
```

so I run `luarocks --lua-version=5.1 --local config variables.LUA 'C:\Users\cc\scoop\apps\lua51\5.1.5\lua5.1.exe'`. and I got:
![image](https://github.com/user-attachments/assets/6291401a-f495-49ce-990e-3752f2e291e0)

escaped `\` in path to: `'C:\\Users\\cc\\scoop\\apps\\lua51\\5.1.5\\lua5.1.exe'`. got the same.

**Comment by KevinNitroG** [comment]
I can reproduce this issue. We can install lua for windows at https://github.com/rjpcomputing/luaforwindows. But I also want to have a better way to resolve this problem.

**Comment by boltlessengineer** [comment]
Unfortunately there is nothing rest.nvim can do about this issue.
This is problem with your package manager, in this case `lazy.nvim`.

I would like to help you but I don’t have access to any Windows machines until next year, so I recommend you to ask for a help from `lazy.nvim` or make an issue about this there. You will get more faster & accurate help in this way.

**Comment by mtfcd** [comment]
> I am able to reproduce this issue. Lua for Windows can be installed via the following link: https://github.com/rjpcomputing/luaforwindows. However, I would prefer to explore a more effective solution to address the problem.

I tried this, did not work. I tried installing lua, lua51 and lua-for-windows use scoop. did not work. I tried use [rocks.nvim](https://github.com/nvim-neorocks/rocks.nvim) to install rest.nvim, and rocks.nvim also could not be installed. But I installed [rocks-scoop](https://github.com/nvim-neorocks/rocks-scoop) as mentioned in [this tutorial](https://github.com/nvim-neorocks/rocks.nvim/wiki/Installing-rocks.nvim-manually,-without-the-installation-script). After that rest.nvim can be isntalled successfully use lazy.nvim.
